### PR TITLE
[DOC] Prepare best practices doc for sharing

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:
Prepares the Best Practices doc for sharing to Grafana Cloud and Grafana by moving the best practices content into a shared file. 

No major doc changes in this PR. 

**Which issue(s) this PR fixes**:
Fixes [#<issue number>](https://github.com/grafana/technical-documentation/issues/710)

**Checklist**
- [ ] Tests updated
- [X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Comment by knylander-grafana** [comment]
@jdbaldry This is the first file that is being shared from Tempo. Do we need to enable anything else? Local build did not show the shared content.

File: docs/sources/tempo/operations/best-practices.md [hunk 1]
```diff
@@ -9,81 +9,7 @@ weight: 20
...
-- Decompose the query
-   - For example, if a complex SQL query involves multiple operations (for example, uses joins, subqueries, or unions), consider creating separate spans for each significant operation.
-- Improve granulation of long-running spans
-     - For long-running operations, you could create a new span for every predetermined interval of execution time.
-        {{% admonition type="note" %}}
-        This requires time-based tracking in your application's code and is more complex to implement.
-        {{% /admonition %}}
-- Use span linking
-     - Should data flow hit bottlenecks where further operations on that data might be batched at a later time, the use of [span links](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md#links-between-spans) can help keep traces constrained to an acceptable time range, while sharing context with other traces that work on the same data. This can also improve the readability of traces.
+{{< docs/shared source="tempo" lookup="best-practices.md" version="latest" >}}
```
Comment (jdbaldry) [inline docs/sources/tempo/operations/best-practices.md:15]:
```suggestion
{{< docs/shared source="tempo" lookup="best-practices-traces.md" version="latest" >}}
```

File: docs/sources/tempo/operations/best-practices.md [hunk 2]
```diff
@@ -9,81 +9,7 @@ weight: 20
 
 This page provides some general best practices for tracing.
 
-## Span and resource attributes
+[//]: # 'Shared content for best practices for traces'
+[//]: # 'This content is located in /tempo/docs/sources/shared/best-practices.md'
```
Comment (knylander-grafana) [inline docs/sources/tempo/operations/best-practices.md:13]:
```suggestion
[//]: # 'This content is located in /tempo/docs/sources/shared/best-practices-traces.md'
```

File: docs/sources/tempo/shared/best-practices-traces.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+---
+headless: true
+labels:
+  products:
```
Comment (knylander-grafana) [inline docs/sources/tempo/shared/best-practices-traces.md:4]:
```suggestion
description: Shared file for tracing best practices.
  products:
```

**Review by zalegrala — APPROVED** [review]

File: docs/sources/tempo/shared/index.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (knylander-grafana) [inline docs/sources/tempo/shared/index.md:1]:
```suggestion
---
description: null
```

File: docs/sources/tempo/shared/index.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+description: null
```
Comment (knylander-grafana) [inline docs/sources/tempo/shared/index.md:2]:
```suggestion
description: No description necessary for a shared file index. 
```

**Pull request merged**